### ISSUE TITLE
Add documentation

### ISSUE THREAD

**Issue by jonc125** [NeurodataWithoutBorders/matnwb#1]
It'd be really helpful to those like me trying to read & understand how the package works to have (Matlab-style) documentation for each class and function. Particularly with the common use of abbreviations for variable & function names, and the lack of comments, it's quite hard to follow the logic!

**Comment by GMCadode** [comment]
I am also currently trying to implement MatNWB, I want to get my data into this format, but some documentation/tutorial would be super-helpful! I've noticed the excellent documentation for pynwb, but does something like this exist for matnwb, or is it in the works? 
Thanks again for this amazing work!

**Comment by bendichter** [comment]
What type data are you working with?

**Comment by GMCadode** [comment]
.mat files mostly, but I can potentially have things in other formats like .csv if necessary

**Comment by bendichter** [comment]
@gmcadode sorry I should have been more specific. Extracellular? Calcium imaging? What acquisition system?

**Comment by GMCadode** [comment]
Oh, gotcha. I'm working with extracellular data, acquired from TDT's system 3 hardware. They have some functions to convert their data format into .mat though, so there's that.

**Comment by bendichter** [comment]
@GMCadode Check out https://github.com/bendichter/to_nwb/blob/master/to_nwb/chang/TDT2NWB.m, which converts TDT data to NWB and relies on `TDTbin2mat.m`. It handles raw LFP and analog (audio) data, but doesn't show you how to incorporate trials, derived data, extensions, etc. I agree we need better documentation but that should give you a good start.

**Comment by GMCadode** [comment]
Thanks! I'll look thru it. Honestly I really believe in what NWB is about, so I'm looking forward to making it work in my lab!

**Comment by ehennestad** [comment]
@bendichter With all the tutorials in place this seems outdated so I close it.
